dm/dm/master/scheduler/scheduler.go

```diff
@@ -2733,44 +2733,32 @@ func (s *Scheduler) handleLoadTask(ctx context.Context, loadTaskCh <-chan ha.Loa
 	}
 }
 
-func (s *Scheduler) OperateValidationTask(expectStage pb.Stage, stCfgs map[string]map[string]config.SubTaskConfig) error {
-	s.mu.RLock()
-	defer s.mu.RUnlock()
+// OperateValidationTask operate validator of subtask.
+// 	tasks: tasks need to operate
+// 	validatorStages: stage info of subtask validators
+// 	changedSubtaskCfgs: length should be 0 or equal to validatorStages
```

Comment:
seems changedSubtaskCfgs can tolerate not the same length as validatorStages 🤔

Reply:
caller should and has make sure about this